server/src/main/java/io/druid/metadata/SQLMetadataConnector.java

```diff
@@ -369,4 +377,37 @@ protected BasicDataSource getDatasource()
     return dataSource;
   }
 
+  private void createAuditTable(final IDBI dbi, final String tableName)
+  {
+    createTable(
+        dbi,
+        tableName,
+        ImmutableList.of(
+            String.format(
+                "CREATE TABLE %1$s (\n"
```

Comment:
id is serial so I assume the intention is you'd filter by audit_key + type and order by id if you want to see a log of changes over time.